Userland/Libraries/LibGfx/PaintStyle.h

```diff
@@ -115,7 +115,7 @@ class RepeatingBitmapPaintStyle : public Gfx::PaintStyle {
 
 class OffsetPaintStyle : public Gfx::PaintStyle {
 public:
-    static ErrorOr<NonnullRefPtr<OffsetPaintStyle>> create(RefPtr<PaintStyle> other, Gfx::AffineTransform transform)
+    static ErrorOr<NonnullRefPtr<OffsetPaintStyle>> create(RefPtr<PaintStyle> other, const Gfx::AffineTransform &transform)
```

Comment:
unfortunately clang-format can't capture this but we use east const notation like `Gfx::AffineTransform const& transform`